src/styles/global.css

```diff
@@ -255,15 +254,11 @@ svg {
   overflow: visible;
 }
 
-
-.team-part {
+<<<<<<< HEAD ======= .team-part {
   color: var(--theme-color);
 }
 
-
-
-
-.wave {
+>>>>>>>3edbfd502a5989e65f242481b6d9a1f2e246735b .wave {
```

Comment:
you can remove this

Reply:
by the way global.css was refactored to global.scss

Reply:
Thank You! I didn’t notice that. I’ll make the changes later and update the PR.

Reply:
> by the way global.css was refactored to global.scss

I can't find it on the latest react_version commit. Did someone remove it?